ci-operator/config/openshift/build-machinery-go/openshift-build-machinery-go-master.yaml

```diff
@@ -0,0 +1,29 @@
+base_images:
+  base:
+    name: "4.4"
+    namespace: ocp
+    tag: base
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.13
+promotion:
+  name: "4.4"
+  namespace: ocp
+resources:
+  '*':
+    limits:
+      memory: 6Gi
+    requests:
+      cpu: "3"
+      memory: 3Gi
+tag_specification:
+  name: "4.4"
+  namespace: ocp
+tests:
+- as: verify
+  commands: |
+    make verify -k --warn-undefined-variables
+  container:
+    from: bin
```

Comment:
Should be `from: src` . Thats why the rehearsal failed.

Reply:
yeah, to have `bin` you need `binary_build_commands`.

Reply:
fixed